src/commands/service/create-revision.test.ts

```diff
@@ -0,0 +1,121 @@
+import os from "os";
+import path from "path";
+import shell from "shelljs";
+import uuid from "uuid/v4";
+import { write } from "../../config";
+import * as azure from "../../lib/git/azure";
+import * as gitutils from "../../lib/gitutils";
+import { IBedrockFile } from "../../types";
+import {
+  getDefaultRings,
+  getSourceBranch,
+  makePullRequest
+} from "./create-revision";
+
+jest
+  .spyOn(gitutils, "getCurrentBranch")
+  .mockReturnValueOnce(Promise.resolve("prod"))
+  .mockReturnValue(Promise.resolve(""));
+const prSpy = jest
+  .spyOn(azure, "createPullRequest")
+  .mockReturnValue(Promise.resolve("done"));
+
+describe("Default rings", () => {
+  test("Get multiple default rings", () => {
+    const randomTmpDir = path.join(os.tmpdir(), uuid());
```

Comment:
prefer to use https://github.com/CatalystCode/spk/blob/master/src/lib/ioUtil.ts#L13
```
import { createTempDir } from ".../../lib/ioUtil.ts";

...
const randomTmpDir = createTempDir();

```